src/fragmenter.c

```diff
@@ -0,0 +1,412 @@
+#include "fragmenter.h"
+#include "toksep.h"
+#include "tokenize.h"
+#include <ctype.h>
+#include <float.h>
+#include <sys/uio.h>
+#include <assert.h>
+
+#define myMin(a, b) (a) < (b) ? (a) : (b)
```

Comment:
should really have these as universal macros somewhere, e.g. RSMin and RSMax?

Reply:
I'm generally using the one in sys/param.h

Reply:
You can just create a minmax.h file with both macros. in util probably.